src/tests/utils/helper.rs

```diff
@@ -21,35 +14,22 @@ pub fn random_bytes() -> [u8; 32] {
 
 pub fn create_key_manager(
     store_keystore_path: &str,
-    store: Rc<Storage>,
+    encrypt: Option<String>,
 ) -> Result<KeyManager, anyhow::Error> {
-    let key_derivation_seed = random_bytes();
-    let winternitz_seed = random_bytes();
+    let random_mnemonic: Mnemonic = Mnemonic::from_entropy(&random_bytes()).unwrap();
 
-    let derivation_path = format!("m/101/1/0/0/{}", generate_random_string());
-    let password = "secret password_123__ABC".to_string();
-    let config = StorageConfig::new(store_keystore_path.to_string(), Some(password));
-    let key_store = Rc::new(Storage::new(&config)?);
-    let keystore = KeyStore::new(key_store);
+    let config = StorageConfig::new(store_keystore_path.to_string(), encrypt);
 
     let key_manager = key_manager::KeyManager::new(
         bitcoin::Network::Regtest,
-        derivation_path.as_str(),
-        Some(key_derivation_seed),
-        Some(winternitz_seed),
-        keystore,
-        store,
+        Some(random_mnemonic),
+        None,
+        &config,
     )?;
 
     Ok(key_manager)
 }
```

Comment:
@TomasSzeSirk @jonasmartin i resolved this conflict, tomy changed password to `secret password_123__ABC`
but this function, now receives the password at parameter `encrypt`

may be we should propagate that to the callers

Reply:
check next commit